accord-core/src/test/java/accord/topology/TopologyRandomizer.java

```diff
@@ -293,7 +300,7 @@ public synchronized Topology updateTopology()
     {
         Topology current = epochs.get(epochs.size() - 1);
         Shard[] oldShards = current.unsafeGetShards().clone();
-        int remainingMutations = random.nextInt(current.size());
+        int remainingMutations = random.nextInt(Math.min(current.size(), 10));
```

Comment:
if `split` keeps happening, then this keeps growing and growing... I hit a case with `current.size() > 300` which caused the test to crawl...

Reply:
Probably shouldn't be a constant and should be a randomized parameter in some range?

Reply:
I don't think a random param helps in our case, it's just a defensive bounds to make sure we don't do too much work.  `split` can cause us to grow, so can `addKeyspace` (my other patch)... so a simple bounds to make sure this method doesn't get out of hands is all I think we really need